datastream-kafka-connector/src/main/java/com/linkedin/datastream/connectors/kafka/AbstractKafkaBasedConnectorTask.java

```diff
@@ -438,11 +438,15 @@ public void run() {
         }
       }
       postShutdownHook();
-      _stoppedLatch.countDown();
+      countDownStoppedLatch();
       _logger.info("{} stopped", _taskName);
     }
   }
 
+  protected void countDownStoppedLatch() {
```

Comment:
@vmaheshw I remember you had expressed concern over this way of doing the count down. IIRC your concern was that any subclass of this can forget to call countDownStopppedLatch (although in this patch Shrinand added a call to it in KafkaMirrorMakerConnectorTask). So just wanted to check if that was considered and concluded that this is the most reasonable way of doing it?
cc @shrinandthakkar

Reply:
Yes, I was not in support of doing it this way, but also did not have a better idea. So, we decided to do it, until we have a better idea.